app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
It is, but it is just called `plan`. Maybe it will be clearer after you pull my updates...if not, ping me